### DISCUSSION TITLE
Expansion Item - active prop

### DISCUSSION THREAD

**Discussion by krychu90** [quasarframework/quasar#14633] [General - Components / Directives / etc]
Hello,
is there any property in `<q-expansion-item>` for active state? `<q-item>` has _active_ prop and _active-class_, but `<q-expansion-item>` has _active-class_ prop only.

**Comment by MilosPaunovic** [comment]
Is there a use case where `active-class` can't help you achieve the requirements?

**Reply by krychu90** [reply]
I have a categories tree with q-item and q-expansion-item (with subcategories). Unfortunately active state by vue router does not work for me. It marks only current category, not parent one. I want to manually set active state. I can do it in q-item but not in q-expansion-item.

**Comment by metalsadman** [comment]
You will have to handle the logic with the router child matching, then you can conditionally add/remove a custom active class in your expansion item using the `header-class` based on the condition.

**Reply by RammsMagnitka86** [reply]
How to add custom class from style section of vue files? It's possible to add custom class only from app.scss for example, but not from style section. Is it bug?
